themes/upbound/layouts/partials/body/left-sidebar-nav.html

```diff
@@ -1,100 +1,94 @@
 <div class="left-sidebar-nav offcanvas-xl offcanvas-start" tabindex="-1" id="left-nav-offcanvas" aria-labelledby="left-nav-offcanvas">
-
     <nav aria-label="Documentation section navigation" class="container px-0" id="left-nav-menu">
         <div class="list-group top-level d-flex">
-            {{/*  First entry for the "Docs" section representing home.html  */}}
             <div class="d-flex flex-row my-0 nav-parent-container position-relative list-group-item list-group-item-action {{if .IsHome }}active{{end}}" id="Docs">
                 <div class="left-offset hide"></div> <!-- Used to keep nav titles aligned evenly by flex -->
                 <a href="{{.Site.BaseURL}}" class="stretched-link">Docs</a>
             </div>
+            {{ $currentPage := . }}
+            {{ with .Site.GetPage "section" "Docs" }}
 
-            {{ $currentPage:= . }}
+            {{ end }}
+            
             {{ range .Site.Sections.ByWeight }}
-
-                {{ $expand := (or (eq $currentPage .) (eq $currentPage.FirstSection .)) }}
-
-                {{/*  Generate a random ID for each parent to use as the collapse target ID  */}}
-                {{ $id := substr (sha1 .Permalink) 0 8 }}
-
-                    <!-- Parent Menu Button -->
-                    {{ if or (not .Page.Params.tocHidden) (eq $.Section "contribute") }}
-                    <div class="d-flex flex-row my-0 nav-parent-container position-relative  {{if eq $currentPage .}}active{{end}}" id="{{.Section}}" data-bs-toggle="collapse" data-bs-target="#collapse-{{$id}}" onclick="this.querySelector('.accordion-button').classList.toggle('collapsed')">
-                        <div class="button-container left-offset">
-                            <div id="{{.Section}}-button" class="mx-0 accordion-button accordion accordion-item {{if not $expand}} collapsed {{end}}" type="button" aria-expanded="false" aria-controls="collapse-{{$id}}" title="Close/Expand"></div>
-                        </div>
-                        <span class="nav-parent-title">{{.Title }}</span>
```

Comment:
Today you can expand and contract the left nav menu without going to the index page. This change makes it so that clicking to expand or contract the menu is the same as clicking on the link. This means an expanded menu can't be closed. 

But this doesn't change the behavior for the child pages, for example here;
https://docs-git-fork-tr0njavolta-main-upboundio.vercel.app/concepts/

Is this intentional?

Reply:
I think it should apply to the child pages as well. The experience I'm trying to address is that people have to click on a side menu item twice to get to any content, I wanted to make those two clicks into one so they see content as soon as possible. 

If we're already creating an index page that no one sees unless they hit the URL directly, we should have some context that users can read from the item instead of going to an "overview" page. It looks like there's some content duplication from that.